.gitignore

```diff
@@ -8,4 +8,5 @@ coverage/
 log/
 .idea/
 *.iml
-.*.sw
+.*.sw*
+.DS_Store
```

Comment:
This will be overwritten. Please open a PR against https://github.com/voxpupuli/modulesync_config/blob/master/moduleroot/.gitignore